lib/Net/DRI/DRD/NGTLD.pm

```diff
@@ -394,6 +394,33 @@ L<Net::DRI::Protocol::EPP::Extensions::CentralNic::Fee> urn:centralnic:params:xm
 =pod
 
 
+=head2 Amazon Registry Services, Inc. (provider: Neustar)
+
+ $dri->add_registry('NGTLD',{provider=>'amazon'});
+
+=head3 Status: Untested
+
+=head3 TLDs
+
+author aws book bot buy call circle fast got jot joy like moi pin read room safe smile song spot talk tunes tushu wanggou xn--1ck2e1b xn--bck1b9a5dre4c xn--cck2b3b xn--eckvdtc9d xn--fct429k xn--gckr3f0f yamaxun you zero
+
+=head3 Custom extensions
+
+L<NET::DRI::Protocol::EPP::Extensions::NeuLevel::Fee> urn:ietf:params:xml:ns:neulevel-1.0
```

Comment:
Based on your FFM one.
I just updated the TLDs (reference: https://ntldstats.com/registry/Amazon-Registry-Services-Inc) and opening with POD Browser looks fine.

The technical documentation that I received mention NeuLevel::Fee as well.